hips/utils/tests/test_healpix.py

```diff
@@ -0,0 +1,15 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from .. healpix import boundaries
+from astropy.coordinates import SkyCoord
+import numpy as np
```

Comment:
In Astropy and most packages this import convention is used (because the `assert_allclose` function is then often called a gadzilliion times, so it's good if it's short.

```
from numpy.testing import assert_allclose
```
Let's use that convention in the `hips` package also.